modules/document_repository/README.md

```diff
@@ -4,10 +4,17 @@
 
  The Document Repository is a tool that provides a centralized location for non-candidate documents that are relevant to the study (i.e. unfilled versions of instruments, publications, or manual of operations).
 
+##  Document Repository Notifications
+
+The Document Repository notifications panel is displayed on the dashboard if a user has been granted either the `document_repository_view` or `document_repository_delete` permission, 
+and displays recently-uploaded documents for current user study. 
+Each document name links to the file for direct download from the Dashboard. 
+Document Repository displays the complete list of available documents.
```

Comment:
There's already:

 "- Interactions: Notifications from the Document Repository module are visible on the Dashboard panel." at the bottom of the document, I believe #6631 was created in error.

Can you just change "- Interactions: " to a heading in the same format as the other READMEs? We don't have top level sections for "Notifications" in any specs.

Reply:
building on @driusan's comment - Just one sentence here describing the notifications feature could replace this section, and then the dashboard widget should be covered below. 
 #6631 is a bit mis-titled but still valid in requesting updates to the test plan etc.